### DISCUSSION TITLE
Generate HTML file for sharing

### DISCUSSION THREAD

**Discussion by KevinKongK** [reata/sqllineage#434] [General]
Very amazing project! It is exactly what I imagine.
However, I'm trying to generate a html file same as the local website in your package as I would like to share to non-tech people.
Is that possible? As LineageRunner holds everything, I think it might be not very difficult? Not very sure...

Thx

**Comment by reata** [comment]
You can download lineage graph for each SQL (mouse over + button and click save).

But it's not possible to download html to share as we require a server for frontent-backend interaction on current implementation.

**Reply by KevinKongK** [reply]
Okay, I see. That could be a substitute. Just want to explain more. In some scenarios the sql script will be very complicate which will present more than 20 links. The highlight function is very important. 

Thanks anyway, so far helps a lot!

**Reply by reata** [reply]
Glad it helped.
